src/commands/app-create.yml

```diff
@@ -14,7 +14,7 @@ parameters:
     description: Shipa team name
     type: env_var_name
   pool:
-    default: SHIPA_POOL
+    default: SHIPA_FRAMEWORK
     description: Shipa pool name
```

Comment:
If you're going to change the environment variable, you need to change the description as well.

Reply:
agree. fixed